live/global.ini

```diff
@@ -62422,7 +62422,7 @@ mobiGlas_MissingBennysDatapad_AboveAndBeyond_Subtitle=Crusader Visitor Informati
d_Artists_Text=/**MOBICONNECT.RETRIEVALSYSTEM.FRP…DATAFOUND**/\n\n•	Denke daran, dass dies ein freier Ausdruck deiner Gedanken ist. Halte dich nicht zurück. Sei ehrlich und du wirst die Wahrheit finden.\n\n•	Wirkung über Substanz oder Transformation über Rehabilitation?\n\n•	Das Medium ist genauso Teil der Diskussion wie die Diskussion selbst ein Medium der Kunst ist, aber wenn diese Botschaft das Werk überwindet, ist das das ultimative Ziel? Sollte das Werk stehen bleiben oder stürzen? Halte inne und frage "Was wäre, wenn", aber stelle sicher, dass die verbleibende Frage immer warum ist.\n\n•	Ich habe eine weitere Kritik gefunden - Obwohl das Werk einen Angriff auf den Raum darstellt, fehlt es dem Angriff an Biss. Die einzige Freundlichkeit in der Erfahrung besteht darin, dass das gesamte übrig gebliebene Harz einer lokalen Kunstschule versprochen wurde. Hoffentlich findet es noch als Kunst Verwendung. Das ist es, was ich brauche. Diesen Schmerz.\n\n•	Während ich anfangs Bedenken hatte, freue ich mich, dass mein neues Quartier sich als fruchtbarer Boden für Inspiration erweist. Das Verlassen des vertrauten Komforts und das Umarmen des Chaos von GrimHEX war so kathartisch, dass ich begonnen habe, den Verlust all meines Geldes als Segen zu betrachten.\n\n•	Du siehst diese anderen Künstler, die "es schaffen". Denara mit seinen Webstühlen, die den Lob der Kritiker aufsaugen, oder Tamsa und ihre bemalten Schiffe. Ihre Umarmung der Kommerzialisierung von Kunst ist das Gift, das unsere Seelen zerstört. Die Tiefgründigkeit dieser Verdinglichung der Menschlichkeit zu enthüllen, wäre eine wahre Inspirationsarbeit.\n\n•	Ich habe heute ein Attentat außerhalb der Basis beobachtet. Es war aufregend und gleichzeitig furchterregend, aber es war das, was danach kam, das mich am meisten beeindruckte. Die Stille, nachdem die Raubtiere davongeflogen waren und die Fracht ihrer Beute gestrippt hatten, ließ mich von dem treibenden Wrack fasziniert sein. Die verdrehten Formen verlieren alle Funktion und werden reine Form. Definitiv ein Motiv, das weiter erforscht werden sollte.\n\n•	Ich habe heute eine Frau getroffen, die sich Threetoe nennt. Sie sagte mir, dass sie, wenn sie sterben würde, von "der Großen Schwester" in den Asteroiden hinausgeschwemmt werden möchte. Sie erklärte, wie ihre Mutter ihr während ihrer Kindheit immer "Ein Geschenk für Baba" vorgelesen habe, und sie habe sich immer mit der Figur verbunden gefühlt. Es erstaunt mich, welchen Einfluss diese Geschichten unserer Jugend auf unser Wesen haben können. Was ersetzt sie, wenn wir älter werden? Die Geschichten hören auf, aber das Bedürfnis bleibt bestehen. Welche Märchen geben uns jetzt Trost? Die Antwort kam zu mir, als meine Gedanken durch Rufe von "So Yum, So Wow" von einem nahegelegenen Big Benny's Automaten unterbrochen wurden. So wird die Geschichte der Menschheit erzählt werden. Dieser korrupte Konsumismus wird das Denkmal sein, das wir hinterlassen.\n\n•	Ein Denkmal, das ich hinterlassen werde...
```

Comment:
Ich habe "Bearbeite nicht" (Don’t edit) einfach entfernt. Keine Ahnung was das heißen soll

`Remember, this is a free flowing expression of your thoughts. Don’t hold back. Don’t edit. Be true and you will find truth.`